jscomp/others/js_option.ml

```diff
@@ -59,5 +59,27 @@ let andThen f x =
   | None -> None 
   | Some x -> f x [@bs]
 
+let map f x =
+  match x with
+  | None -> None
+  | Some x -> Some (f x [@bs])
```

Comment:
> Could just do both though. :-)
I was going to say that... maybe we should implement both the standard library signatures and have aliases that are friendlier to people coming from javascript?